doc/classes/bool.xml

```diff
@@ -42,7 +42,7 @@
 		}
 		[/csharp]
 		[/codeblocks]
-		[b]Note:[/b] In modern programming languages, logical operators are evaluated in order. All remaining conditions are skipped if their result would have no effect on the final value. This concept is known as [url=https://en.wikipedia.org/wiki/Short-circuit_evaluation]short-circuit evaluation[/url] and can be useful to avoid evaluating expensive conditions in some performance-critical cases.
+		[b]Note:[/b] In modern programming languages, logical operators are evaluated in order. All remaining conditions are skipped if their result would have no effect on the final value. This concept is known as [url=https://en.wikipedia.org/wiki/Short-circuit_evaluation]short-circuit evaluation[/url] and can be useful to avoid evaluating expensive conditions in some performance-intensive cases.
```

Comment:
This is not a 1 to 1 replacement, performance critical doesn't mean that it's expensive or intensive, but that the performance being good is critical, I wouldn't change this, and "performance critical" is a *very* well established term, whereas "performance intensive" isn't really

Reply:
I was the one that wrote that last time. In that case, "performance-critical" should be used more frequently across the reference, but finding where may be problematic.